index.html

```diff
@@ -1,12 +1,259 @@
+            </p>
+          </li>
+          <li>
+            <span class="number">02</span>
+            <img
+              src="./images/icon_2.svg"
+              alt="щит із зображенням лупи всередині"
+            />
+            <h3>Secure</h3>
+            <p class="technology_text">
+              All your information is stored on secure cloud servers
+            </p>
+          </li>
+          <li>
+            <span class="number">03</span>
+            <img src="./images/icon_3.svg" alt="зображення пдф файлу" />
+            <h3>PDF Download</h3>
+            <p class="technology_text">
+              Download any of your reports in PDF format
+            </p>
+          </li>
+          <li>
+            <span class="number">04</span>
+            <img src="./images/icon_4.svg" alt="зображення сіесве файлу" />
+            <h3>CSV Download</h3>
+            <p class="technology_text">
+              All your information is stored on secure cloud servers
+            </p>
+          </li>
+        </ul>
+      </section>
+      <section class="section_4">
+        <h2>Stay focused on saving money</h2>
+        <p class="stay_text">
+          It is important to stay focused on saving money in any way you can. We
+          help you monitor your spending habits so you can easily spot and cut
+          any unnecessary expenses. Simply join today to get started!
+        </p>
+        <a href=""
+          ><img
+            src="./images/link_icon_1.svg"
+            alt="іконка-посилання на плеймаркет"
+        /></a>
+        <a href=""
+          ><img src="./images/link_icon_2.svg" alt="іконка-посилання"
+        /></a>
+        <a href=""
+          ><img
+            src="./images/link_icon_3.svg"
+            alt="іконка-посилання на сайт майкрософт"
+        /></a>
+        <figure>
+          <iframe
+            src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2645.5055203692987!2d35.05302421554022!3d48.46601637925102!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0%3A0x9ffb0c281aeab08e!2z0JrQvtC80L_RjNGO0YLQtdGA0L3QsNGPINGI0LrQvtC70LAgSGlsbGVs!5e0!3m2!1sru!2sua!4v1606076668933!5m2!1sru!2sua"
+            width="600"
+            height="450"
+            style="border: 0"
+            allowfullscreen=""
+            aria-hidden="false"
+            tabindex="0"
+          ></iframe>
+        </figure>
+      </section>
+      <section class="section_1">
+        <h2>Contact</h2>
+        <p class="stay_text">
+          Questions or concerns? Just fill out the form below and our support
+          team will get back to you within 24 hours. But I must explain to you
+          how all this mistaken idea of denouncing pleasure and praising pain
+          was born and I will give you a complete account of the system, and
+          expound the actual teachings of the great explorer of the truth, the
+          master-builder of human happiness.
+        </p>
+        <form>
+          <label
```

Comment:
why do you need label at this point? remove.

Reply:
Ok, removed.